index.src.html

```diff
@@ -1591,9 +1594,10 @@ <h4 algorithm id="construct-federatedcredential-data">
   mediated=] actions will involve presenting the user some sort of UI, and asking them to make a
   decision.
 
-  An action is <dfn export>unmediated</dfn> if it takes place without explicit user consent. For
-  example, if a user configures their browser to grant persistent credential access to a particular
-  origin, credentials may be provided without presenting the user with a UI requesting a decision.
+  An action is <dfn export>unmediated</dfn> or <dfn export>silent</dfn> if it takes place without
+  explicit user consent. For example, if a user configures their browser to grant persistent
```

Comment:
We never use this definition, and I don't actually think it makes sense to export something this generic. How about replacing the first sentence with:

"""
An action is unmediated if it takes place silently, without explicit user consent.
"""